ci/cases/yamls/gfs_defaults_ci.yaml

```diff
@@ -1,6 +1,14 @@
+defaults:
+  !INC {{ HOMEgfs }}/parm/config/gfs/yaml/defaults.yaml
+base:
+  ACCOUNT: {{ 'HPC_ACCOUNT' | getenv }}
+  DO_TEST_MODE: "YES"
+
```

Comment:
There are two sets of "defaults" and "base" in this yaml.  I think the first one should be removed.

```suggestion
```

Reply:
Lines 1-6 are still present.

Reply:
Apologies, please disregard.